app/app.go

```diff
@@ -259,6 +318,8 @@ func NewDesmosApp(
 
 		// Custom modules
 		profilestypes.StoreKey,
+
+		wasm.StoreKey,
```

Comment:
As above, unless strictly necessary, I would put this after the `authzkeeper.StoreKey`